aws/cloudformation/bootstrap_chef_stack.sh.erb

```diff
@@ -39,6 +39,12 @@ cat <<JSON > $FIRST_BOOT
     "build_apps": true,
     "use_my_apps": true,
 <% end -%>
+<% if @frontends -%>
+    "memcached_endpoint": "${Memcached.ConfigurationEndpoint.Address}:${Memcached.ConfigurationEndpoint.Port}",
+    "redis_primary": "${GeocoderGroup.PrimaryEndPoint.Address}:${GeocoderGroup.PrimaryEndPoint.Port}",
+    "redis_read_addresses": "${GeocoderGroup.ReadEndPoint.Addresses}",
+    "redis_read_ports": "${GeocoderGroup.ReadEndPoint.Ports}",
```

Comment:
Added these `redis_*` properties as well so that the Redis instance can also be automatically configured within the application (it is currently hooked up through manually-configured Chef secrets).